ios/A0Auth0.m

```diff
@@ -98,6 +106,10 @@ - (void)presentAuthenticationSession:(NSURL *)url {
                                           }
                                           self.authenticationSession = nil;
                                       }];
+        #if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
```

Comment:
Change to 
`if (@available(iOS 13.0, *)) {`
To prevent warning

Reply:
Actually, I just tried this on iOS 12.4, and is fails with this suggested change.
Looks like it may need to stay how you originally had it and live with the warning.

Reply:
If you try and compile in Xcode 10.x only with @available it will error as the compiler still sees the code.